usagov/usagov-2021
# Usagov 1627 Front end tweaks from demo feedback

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1627

## Description

* Fixes buttons to use sentence casing
* Updates translation for "Life Events" on Spanish page
* Use short description field instead of page intro if the former has content
* Add fields for customizing the button label on call out boxes.
## Type of Changes


On English search results
![image](https://github.com/usagov/usagov-2021/assets/493854/ada25112-c830-4110-a8ec-1fab99ca64d3)

On Spanish search results (search discapacidad)
![image](https://github.com/usagov/usagov-2021/assets/493854/d0ce0907-5208-4717-a76e-7a83af58f765)

On benefits topic landing page
![image](https://github.com/usagov/usagov-2021/assets/493854/c35d7e71-4945-4971-b938-18aefe802116)

On homepage, in callout field group

![image](https://github.com/usagov/usagov-2021/assets/493854/65c673c5-a312-4a29-8b30-b4f1761ac1a7)


## Testing Instructions



- Confirm case on english buttons is correct
- Confirm category for life events in spanish results
- Search results render with short description fields
- Edit forms have field for button text on home page and benefits topic page

### Requires New Config

### Requires New Content

### Validation Steps


## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
